f2/src/forms/LocationInfoForm.js

```diff
@@ -39,7 +36,7 @@ export const LocationInfoForm = props => {
                   </Trans>
                 </FormHelperText>
                 <TextInput
-                  id="email"
+                  id="postalCode"
```

Comment:
I think you might have branched this off your other PR branch rather than master.

Reply:
Gone!